lib/core/basebandApiManager.cpp

```diff
@@ -201,13 +208,13 @@ void basebandApiManager::handle_request_callback(connectionInstance& conn, json&
         if (start_unix_seconds >= 0) {
             int64_t start_unix_nano = request["start_unix_nano"];
             struct timespec ts = {start_unix_seconds, start_unix_nano};
-            start_fpga = compute_fpga_seq(ts);
+            start_fpga = tel.to_seq(ts);
         } else {
             start_fpga = -1;
         }
 
         int64_t duration_nano = request["duration_nano"];
-        int64_t duration_fpga = duration_nano / FPGA_PERIOD_NS;
+        int64_t duration_fpga = (int64_t)(duration_nano / (1e9 * ts_to_double(tel.seq_length())));
```

Comment:
Isn't there `tel.seq_length_ns()` that could be used directly (just for better readability)?

Reply:
Yeah there is. That function didn't actually exist when I ported this file. I'll go back and change it.